README

```diff
@@ -93,8 +99,9 @@ application.  The QDP and MILC interfaces can be disabled with the,
 e.g., --disable-milc-interface flag.
 
 The eigen-vector solvers (eigCG and incremental eigCG) require the
-installation of the MAGMA dense linear algebra package.  Supported
-versions are MAGMA 1.5.x and 1.6.x, available from
+installation of the MAGMA dense linear algebra package.  It is
+recommended that MAGMA 1.7.x is used, though versions are 1.5.x and
+1.6.x should work.  MAGMA is available from
 http://icl.cs.utk.edu/magma/index.html.  MAGMA is enabled using the
 configure option --with-magma=MAGMA_PATH.
```

Comment:
Yes.  Once you've finished the cmake wiki page we can update this as is needed.  Perhaps we don't need to have so much detail in the README and just refer to the wiki?